src/requirements/data/majors/ece.ts

```diff
@@ -103,14 +75,13 @@ const eceRequirements: readonly CollegeOrMajorRequirement[] = [
         !(ifCodeMatch(course.catalogNbr, '1***') || ifCodeMatch(course.catalogNbr, '2***')),
     ],
     fulfilledBy: 'courses',
-    perSlotMinCount: [1],
+    perSlotMinCount: [3],
```

Comment:
This change to make this 3 courses, and the one below to make it 2, LGTM! I do think they also need to be toggleable though as well as this was a 2021 change. Could you implement that please?